### PR TITLE
Update README.md ( docs first page)

### PR DESCRIPTION
**Pull request by Lavanyagaur22**
Update README.md (docs)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <img width="400" src="https://github.com/aerogear/offix/raw/master/resources/logo.png">
   <br/>
   Offix extends the capabilities of Apollo GraphQL providing a
-  fully featured Offline Workflow and Conflict Resolution.
+  fully featured Offline Workflow and Conflict Resolution for Android Platform.
 </p>
```
Comment (harshithdwivedi) [inline docs/README.md:7]: Add a link to the docs below the install header

File: docs/README.md [hunk 2]
```diff
@@ -12,12 +12,26 @@ It will leverage Apollo GraphQL cache do deliver seamless experience for queryin
...
-- Offline Listeners and workflows for seamless UI.
-- Flexible, out of the box Conflict Resolution implementations
-- Subscriptions and Binary Upload that works offline.
-- Multi platform support.
-    - Works with Web, Cordova, Capacitor and React Native.
-- Framework agnostic (works with React, Angular and Vue)
+### Offline support
+- Mutations are persisted when Offline and replicated back to server when online.
+- Offline Listeners for performing UI Bindings according to the requirements.
+
```
Comment (harshithdwivedi) [inline docs/README.md:40]: Add an option for installing the library

**Comment by harshithdwivedi** [comment]
Have a look at the comments

**Review by harshithdwivedi — CHANGES_REQUESTED** [review]

**Comment by Lavanyagaur22** [comment]
@harshithdwivedi This readme will be the first page of our docs.

**Review by harshithdwivedi — APPROVED** [review]

**Pull request merged**